### ISSUE TITLE
Request: SVGO playground on the website

### ISSUE THREAD

**Issue by KTibow** [svg/svgo#1845]
SVGOMG doesn't get any updates anymore. Add a playground to svgo.dev so that you can minify an SVG with SVGO there.

**Comment by TrySound** [comment]
Or we could just contribute to svgomg. It's not like completely unsupported.

**Comment by KTibow** [comment]
Its dependencies are out of date. Despite there being PRs that bring it back to up to date, it hasn't been updated for half a year

**Comment by TrySound** [comment]
Those deps do not affect user in any way. SVGO itself was not actively maintained for some time. Thanks to @SethFalco for bringing it back to life.

**Comment by KTibow** [comment]
Fair enough. However I still think that it would be better to transition to svgo.dev for the same reasons as before. I'm going to be away from my computer but I might try to implement this when I get back

**Comment by SethFalco** [comment]
I was exploring if we should do something like this, but opted against it because there are other clients already. However, it's true for SVGOMG at least that maintenance is lacking.

* [SVGOMG](https://svgomg.net/), which was already discussed here
* [OhMySVG](https://github.com/sonnyp/OhMySVG)

For now, I'll transfer this to svg/svgo.dev, but I'm not prepared to commit to maintaining a playground yet. We can open room for discussion once svg/svgo has it's pending issues and pull requests back on track, though.

In other words, SVGO should perform it's current objectives well before creating more maintenance burden.
